packages/common/src/functions-api.ts

```diff
@@ -62,17 +65,23 @@ export interface GetDonorAppointmentsResponse {
   futureAppointments: BookedAppointmentApiEntry[];
 }
 
-export const BookAppointmentFunctionName = "bookAppointment";
+export const CoordinatorBookAppointmentFunctionName =
+  "coordinatorBookAppointment";
+export const DonorBookAppointmentFunctionName = "bookAppointment";
```

Comment:
Move this down next to the function request object

Reply:
it is next to the request object